cmd/osbuild-composer-dbjobqueue-tests/main_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
 // +build integration
```

Comment:
I think my JetBrains IDE did this, can I leave this? I am assuming this is some kind of new syntax?